### ISSUE TITLE
Fix the unity trees

### ISSUE THREAD

**Issue by FredrikNoren** [AmbientRun/Ambient#983]
This is what they should look like:
![Screenshot 2023-09-27 at 19 05 55](https://github.com/AmbientRun/Ambient/assets/1288009/f5688d6d-c82d-4122-bc88-385e946e4ddd)

This is what they are looking like currently:
![Screenshot 2023-09-27 at 18 54 37](https://github.com/AmbientRun/Ambient/assets/1288009/19874e29-aabb-41b8-a816-3fda5a6ca29f)

I see at least three problems:
1. The needles are way to thin
2. All lods are rendered
3. One of the lods is 90 degrees off

**Comment by ten3roberts** [comment]
You mentioned that you managed to find the cause of the needles white outline? @FredrikNoren 

Could you write it down here and the relevant branch?

**Comment by FredrikNoren** [comment]
@ten3roberts Yeah the white outlines most likely came from the background of the webpage "shining through". Our shaders are writing alpha values, so the entire canvas on the web becomes semi-transparent. I had the same problem with the golden images (their alpha values were not always 255), so had to set alpha to 255 on them everywhere.

We should restore the alpha cutoff to whatever it was before as well.
